src/locale/ids/fr.ts

```diff
@@ -0,0 +1,184 @@
+import { Locale } from "..";
+
+export const en: Locale = {
```

Comment:
```suggestion
export const fr: Locale = {
```

@sebjulliand You forgot to change the locale code from `en` to `fr`.

You must also add the `fr` locale code to the files `src/locale/index.ts` and `package.json`...

Reply:
Ugh, I guess I was way to excited to ship the translations 😅